qiskit/transpiler/passes/utils/gate_direction.py

```diff
@@ -99,10 +103,10 @@ def run(self, dag):
             control = node.qargs[0]
             target = node.qargs[1]
 
-            physical_q0 = trivial_layout[control]
-            physical_q1 = trivial_layout[target]
+            physical_q0 = trivial_layout.v2p[control]
+            physical_q1 = trivial_layout.v2p[target]
 
-            if self.coupling_map.distance(physical_q0, physical_q1) != 1:
+            if self.coupling_map.dist_matrix[physical_q0][physical_q1] != 1:
```

Comment:
indexing

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/7036/commits/9268df5513b4965ecf2ea457c8bf86ba697586f1